service/lib/agama/dbus/storage/interfaces/device/lvm_vg.rb

```diff
@@ -75,7 +68,6 @@ def lvm_vg_lvs
             def self.included(base)
               base.class_eval do
                 dbus_interface VOLUME_GROUP_INTERFACE do
-                  dbus_reader :lvm_vg_name, "s", dbus_name: "Name"
```

Comment:
Note this name is not the same than the device name. For example, the device name could be `/dev/vg0` and this would be `vg0`

Reply:
Actually it uses the device name.